src/docs/product/alerts-notifications/issue-alerts.mdx

```diff
@@ -4,7 +4,9 @@ sidebar_order: 2
 description: "Learn how to configure and manage Issue Alerts."
 ---
 
-Issue alerts apply only to error events, and trigger whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users. Sentry users with admin permissions or higher can create alerts. To confirm or set admin permissions, navigate to **Project Settings > Alerts > New Alert**. To change the frequency of your alerts, configure your [Alert Settings](/product/alerts-notifications/alert-settings/).
+Issue alerts apply only to error events, and trigger whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users.
```

Comment:
Are we capitalizing Issue Alerts and Metric Alerts? We seem to be inconsistent.

Reply:
we shouldn't be capitalizing Alerts. We discussed capitalizing Issue because we use "'issue" as a synonym for "problem" sometimes. But I don't have a good suggestion here, I'd err on the side of not capitalizing.